framework/source/class/qx/ui/list/List.js

```diff
@@ -565,6 +581,18 @@ qx.Class.define("qx.ui.list.List",
     },
 
 
+    // property apply
+    _applyVariableItemHeight : function(value, old) {
+      if(value) {
```

Comment:
Property apply should not be called when value == old. Except you do it manually.